code/game/gamemodes/objective.dm

```diff
@@ -348,6 +355,16 @@
 			return FALSE
 	return TRUE
 
+/datum/objective/exist //Like survive, but works for silicons and zombies and such.
```

Comment:
This entire datum could just be 1 proc

/datum/objective/survive/exist/considered_alive()

The 'survive' wording is already understandable enough